src/CreateLesson/CreateLesson.js

```diff
@@ -0,0 +1,33 @@
+import React, {useState} from 'react'
+import './CreateLesson.scss'
+import QuestionFormCard from '../QuestionFormCard/QuestionFormCard'
+import LessonPreview from '../LessonPreview/LessonPreview'
+
+const CreateLesson = () => {
+  const [questions, setQuestions] = useState([])
+  const [lessonTitleText, setLessonTitleText] = useState("");
+  
```

Comment:
CreateLesson is the parent component that allows for the questions to be created & saved via hooks, and then to display them on the right side of the screen for a teacher.